lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
@oschreib Yes I do. `def proxy_url_service(namespace, service, port, protocol)` makes more sense to me, only adding in to URL if port & protocol are specified.